addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -288,8 +288,8 @@ QUnit.test('mark channel as fetched when a new message is loaded and as seen whe
                 name: "Fred",
             },
         ]]],
-        message_unread_counter: 1, // seen would not be called if not > 0
         model: 'mail.channel',
+        serverMessageUnreadCounter: 1, // seen would not be called if not > 0
```

Comment:
ideally these tests should be converted to use this.data, also few lines above there is a fixme and the related task is resolved

Reply:
Yeah but there is a lot of tests to change so they use data, should they be done in this PR ?

Reply:
No let's not delay it for that. Would be nice it we could make a pass later though, maybe just create a task too.

Reply:
https://www.odoo.com/web#id=2336360&action=333&active_id=1750&model=project.task&view_type=form&cids=1&menu_id=4720